util/chplenv/chpl_gpu.py

```diff
@@ -24,6 +24,4 @@ def get_cuda_path():
         return ""
 
 def validate(chplLocaleModel, chplComm):
-    if chplLocaleModel == 'gpu' and chplComm != "none":
-        error("The prototype GPU support does not work when CHPL_COMM is not set to\n\"none\".");
-
+    pass
```

Comment:
Should we make this fail for ugni? Should we test this PR on an XC with GPUs?

(Can happen post-merge)

Reply:
Ah, didn't think of that. I had a sense we'd want to keep that validation function around.  I've modified it to this:

```python
def validate(chplLocaleModel, chplComm):
    validModels = ["none", "gasnet"]
    if chplLocaleModel == 'gpu' and chplComm not in ["none", "gasnet"]:
      error("The prototype GPU support does not work when CHPL_COMM is not set to one of:\n\t%s." %
         ", ".join(validModels))
```